src/hooks/useMetaMask.ts

```diff
@@ -14,16 +12,9 @@ export type MetaMaskInfo = {
 * returns boolean metaMask exists at all
 */
 
-export default function useSelectedProvider():MetaMaskInfo {
+export default function useMetaMask():MetaMaskInfo {
```

Comment:
check if still required

Reply:
There is a single place where useMetaMask is still relevant (I think).

it's for the sideBar where we .request 'watch_asset' and I decided to keep using the ethereum object instead of the eipProvider / library provider.

because of type incompatibility between .request and our token param in the request.
no easy way to override this specific request, without having to override all their request definitions